HACKING.rst.txt

```diff
@@ -194,289 +219,6 @@ project and SciPy the community, and it can always use a new pair of hands.
-
-Now, whenever you want to switch to the virtual environment, you can use the
-command ``source scipy-dev/bin/activate``, and ``deactivate`` to exit from the
-virtual environment and back to your previous shell.  With scipy-dev
-activated, install first Scipy's dependencies::
-
-    $ pip install NumPy pytest Cython
-
-After that, you can install a development version of Scipy, for example via::
-
-    $ python setup.py install
-
-The installation goes to the virtual environment.
-
-
-*How do I set up an in-place build for development*
-
-For development, you can set up an in-place build so that changes made to
-``.py`` files have effect without rebuild. First, run::
-
-    $ python setup.py build_ext -i
-
-Then you need to point your PYTHONPATH environment variable to this directory.
-Some IDEs (`Spyder`_ for example) have utilities to manage PYTHONPATH.  On Linux
-and OSX, you can run the command::
-
-    $ export PYTHONPATH=$PWD
-
-and on Windows
-
-    $ set PYTHONPATH=/path/to/scipy
-
-Now editing a Python source file in SciPy allows you to immediately
-test and use your changes (in ``.py`` files), by simply restarting the
-interpreter.
-
-
-*Are there any video examples for installing from source, setting up a
-development environment, etc...?*
-
-Currently, there are two video demonstrations for Anaconda Python on macOS:
-
-`Anaconda SciPy Dev Part I (macOS)`_ is a four-minute
-overview of installing Anaconda, building SciPy from source, and testing
-changes made to SciPy from the `Spyder`_ IDE.
-
-`Anaconda SciPy Dev Part II (macOS)`_ shows how to use
-a virtual environment to easily switch between the "pre-built version" of SciPy
-installed with Anaconda and your "source-built version" of SciPy created
-according to Part I.
-
-
-*Are there any video examples of the basic development workflow?*
-
-`SciPy Development Workflow`_ is a five-minute example of fixing a bug and
-submitting a pull request. While it's intended as a followup to
-`Anaconda SciPy Dev Part I (macOS)`_ and `Anaconda SciPy Dev Part II (macOS)`_,
-the process is similar for other development setups.
-
-
-*Can I use a programming language other than Python to speed up my code?*
-
-Yes.  The languages used in SciPy are Python, Cython, C, C++ and Fortran.  All
-of these have their pros and cons.  If Python really doesn't offer enough
-performance, one of those languages can be used.  Important concerns when
-using compiled languages are maintainability and portability.  For
-maintainability, Cython is clearly preferred over C/C++/Fortran.  Cython and C
-are more portable than C++/Fortran.  A lot of the existing C and Fortran code
-in SciPy is older, battle-tested code that was only wrapped in (but not
-specifically written for) Python/SciPy.  Therefore the basic advice is: use
-Cython.  If there's specific reasons why C/C++/Fortran should be preferred,
-please discuss those reasons first.
-
-
-*How do I debug code written in C/C++/Fortran inside Scipy?*
```

Comment:
Moved to [`doc\source\dev\contributor\compiled_code.rst`](https://github.com/scipy/scipy/pull/10286/files#diff-0277b675f933e117e69eaf469d0dbb37) linked from CTOC.